app/api/admin_statistics_api/events.py

```diff
@@ -50,3 +53,405 @@ class AdminStatisticsEventDetail(ResourceDetail):
         'class': NoModelLayer,
         'session': db.session
     }
+
+
+class AdminStatisticsEventTypeSchema(Schema):
+    """
+    Api schema
+    """
+    class Meta:
+        """
+        Meta class
+        """
+        type_ = 'admin-statistics-event-type'
+        self_view = 'v1.admin_statistics_event_type_detail'
+        inflect = dasherize
+
+    id = fields.String()
+    camp_treat_retreat = fields.Method("camp_treat_retreat_count")
+    appearance_or_signing = fields.Method("appearance_or_signing_count")
+    attraction = fields.Method("attraction_count")
+    class_training_or_workshop = fields.Method("class_training_or_workshop_count")
+    concert_or_performance = fields.Method("concert_or_performance_count")
+    conference = fields.Method("conference_count")
+    convention = fields.Method("convention_count")
+    dinner_or_gala = fields.Method("dinner_or_gala_count")
+
+    festival_or_fair = fields.Method("festival_or_fair_count")
+    game_or_competition = fields.Method("game_or_competition_count")
+    meeting_or_networking_event = fields.Method("meeting_or_networking_event_count")
+    party_or_social_gathering = fields.Method("party_or_social_gathering_count")
+    race_or_endurance_event = fields.Method("race_or_endurance_event_count")
+
+    rally = fields.Method("rally_count")
+    screening = fields.Method("screening_count")
+    seminar_or_talk = fields.Method("seminar_or_talk_count")
+    tour = fields.Method("tour_count")
+    tournament = fields.Method("tournament_count")
+    other = fields.Method("other_count")
+
+    @staticmethod
+    def camp_treat_retreat_count(obj):
```

Comment:
Can you explain why this method needs to be static ? If we look at the other Schema classes like ```AdminStatisticsEventSchema```, they aren't static.

Reply:
@srv-twry I have made it so because earlier codacy reported these error as to 
`Method could be a function`

@niranjan94 So should I retain it as `@staticmethod` or define the methods without `@staticmethod`

Reply:
Static method don't receive `self` while the marshmallow's definition provides it. Therefore if the interface of marshmallow doesn't declare it as static then don't define it as static.